.foreman

```diff
@@ -0,0 +1 @@
+env: .env
```

Comment:
**question, non-blocking**: Does this just make [the default behavior](https://share.getcloudapp.com/JruGX5Zp) explicit?

Reply:
@anandaroop - good catch! i just tried `foreman start -f Procfile.dev web` again, w/o `.foreman` file, and it works! it didn't work yesterday but now i suspect it was likely due to something else. i'll pr removing `.foreman` file.